main/src/io/github/iltotore/iron/constraint/string.scala

```diff
@@ -84,3 +89,15 @@ object string:
       (valueExpr.value, regexExpr.value) match
         case (Some(value), Some(regex)) => Expr(value.matches(regex))
         case _                          => '{ $valueExpr.matches($regexExpr) }
+
+  object NotBlank:
+    inline given Constraint[String, NotBlank] with
+
+      override inline def test(value: String): Boolean = ${ checkNotBlank('value) }
+
+      override inline def message: String = "Should contain at least 1 non-whitespace character"
+
+    def checkNotBlank(valueExpr: Expr[String])(using Quotes): Expr[Boolean] =
```

Comment:
`checkNotBlank` should be private.

Reply:
Suggestion: this and other similar methods could now be renamed to `check` since they are within a companion object. I agree this should be private, but I would leave `Match.check` public as it currently is, as it's useful to define custom instances and re-use the macro check for compile-time validation.

Reply:
Good point. I changed the method name and made it private. I haven't touched the methods for other objects though